fixtures/flight/server/handler.server.js

```diff
@@ -10,10 +11,13 @@ function resolve(path) {
 }
 
 module.exports = async function(req, res) {
+  CacheContext._currentValue = createCache();
```

Comment:
Essentially a global now. But it should probably be created by the server itself (?)